src/ci/shared.sh

```diff
@@ -13,7 +13,9 @@ function retry {
   local n=1
   local max=5
   while true; do
-    "$@" && break || {
+    if "$@"; then
+      break
+    else
       if [[ $n -lt $max ]]; then
```

Comment:
```suggestion
    elif [[ $n -lt $max ]]; then
```
and also fix the indentation